src/main/java/org/springframework/data/jpa/repository/query/CriteriaQueryParameterBinder.java

```diff
@@ -60,19 +59,18 @@
 	 */
 	@Override
 	@SuppressWarnings("unchecked")
-	protected void bind(Query query, JpaParameter parameter, Object value, int position) {
+	protected QueryParameterSetter bind(JpaParameter parameter, Object value, Integer position) {
```

Comment:
The parameter "position" won't be used here? And why is it changed to Integer? It's only (at least within this context) invoked with primitive values, so it's boxed for nothing?